src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/EmptyLineSeparatorCheck.java

```diff
@@ -458,13 +459,8 @@ private static boolean isClassMemberBlock(int astType) {
      * @return true if the token has not allowed multiple empty lines before.
      */
     private boolean hasMultipleLinesBefore(DetailAST ast) {
-        boolean result = false;
-        if ((ast.getType() != TokenTypes.VARIABLE_DEF
-            || isTypeField(ast))
-                && hasNotAllowedTwoEmptyLinesBefore(ast)) {
-            result = true;
-        }
-        return result;
+        return (ast.getType() != TokenTypes.VARIABLE_DEF || isTypeField(ast))
+                    && hasNotAllowedTwoEmptyLinesBefore(ast);
```

Comment:
if this is just refactoring, please move to separate commit (without reference  to issue in commit name)

Reply:
done

Reply:
done